src/Microsoft.AspNetCore.Razor.Language/Resources.resx

```diff
@@ -264,4 +264,34 @@
   <data name="InvalidRazorLanguageVersion" xml:space="preserve">
     <value>Provided value for razor language version is unsupported or invalid: '{0}'.</value>
   </data>
+  <data name="AddTagHelperDirective_StringToken_Description" xml:space="preserve">
+    <value>Represents the tag helper type name and the assembly name to add the tag helpers from. Specify '*' for type name to include all tag helper types from the given assembly.</value>
+  </data>
+  <data name="AddTagHelperDirective_StringToken_Name" xml:space="preserve">
+    <value>TypeName, AssemblyName</value>
+  </data>
+  <data name="InheritsDirective_TypeToken_Description" xml:space="preserve">
+    <value>The base type that the current page inherits.</value>
+  </data>
+  <data name="InheritsDirective_TypeToken_Name" xml:space="preserve">
+    <value>TypeName</value>
+  </data>
+  <data name="RemoveTagHelperDirective_StringToken_Description" xml:space="preserve">
+    <value>Represents the tag helper type name and the assembly name to remove the tag helpers from. Specify '*' for type name to include all tag helper types from the given assembly.</value>
```

Comment:
`Remove tag helpers from the specified type name and assembly name. Specify '*' for the type name to remove all tag helper types from the specified assembly.`